src/lighteval/metrics/llm_as_judge.py

```diff
@@ -112,6 +111,11 @@ def evaluate_answer(
         Raises:
             Exception: If an error occurs during the API call.
         """
+        if self.client is None:
+            from openai import OpenAI
```

Comment:
You can also add a test here btw but it should be redundant with one in lighteval_task